Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/application/EndpointManager.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2009-2014, United States Government, as represented by the Secretary of Health and Human Services.
```

Comment:
I think this is out of scope.  We should have a task to update all copyrights to 2015.

From: Christopher May [mailto:notifications@github.com]
Sent: Tuesday, January 13, 2015 11:48 AM
To: CONNECT-Solution/CONNECT
Cc: Smith, Jason A (CGI Federal)
Subject: Re: [CONNECT] FHAC-94 Admin GUI Connection Manager (#994)


In Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/application/EndpointManager.java<https://github.com/CONNECT-Solution/CONNECT/pull/994#discussion-diff-22875209>:

> @@ -0,0 +1,47 @@

> +/*

> + * Copyright (c) 2009-2014, United States Government, as represented by the Secretary of Health and Human Services.

Do we need to update the copyright on new files to 2015?

—
Reply to this email directly or view it on GitHub<https://github.com/CONNECT-Solution/CONNECT/pull/994/files#r22875209>.